github-actions/src/main/scala/org/typelevel/sbt/gha/GenerativePlugin.scala

```diff
@@ -514,6 +521,7 @@ ${indent(jobs.map(compileJob(_, sbt)).mkString("\n\n"), 1)}
     githubWorkflowBuildMatrixInclusions := Seq(),
     githubWorkflowBuildMatrixExclusions := Seq(),
     githubWorkflowBuildRunsOnExtraLabels := Seq(),
+    githubWorkflowBuildTimeoutMinutes := None,
```

Comment:
I wonder if we should make the default `Some(60)`? I think that is sufficient for all Typelevel projects. It would would prevent a hanging build in Cats Effect from taking up 24 runners for 6 hours 😅

Reply:
Makes sense to me